cryptol-remote-api/python/cryptol/__init__.py

```diff
@@ -2,8 +2,17 @@
 
 from __future__ import annotations
 
-from typing import NoReturn
-
+import base64
+import os
+from enum import Enum
+from dataclasses import dataclass
+from distutils.spawn import find_executable
+from typing import Any, List, NoReturn, Optional, Union
+from typing_extensions import Literal
+
+import argo_client.interaction as argo
+from argo_client.interaction import HasProtocolState
+from argo_client.connection import DynamicSocketProcess, ServerConnection, ServerProcess, StdIOProcess, HttpProcess
```

Comment:
Does adding these `argo_client` imports and others like them back in actually do anything for users of this package? E.g., if someone has `import cryptol`, does that make any of these names in-scope and/or accessible?

Reply:
From what I can tell, yes. If you `import cryptol` then you can refer to `cryptol.find_executable`, for example, since the line `from distutils.spawn import find_executable` is in `__init__.py`.

Of course, it would be gross to actually do this – but I realized I couldn't claim "no scripts should have to be updated" unless I kept these lines. If we're fine with the small chance of breaking someone's script, it would make more sense to delete them.